python_transport/wirepas_gateway/dbus/c-extension/dbus_c.c

```diff
@@ -29,16 +27,8 @@ static int on_packet_received(sd_bus_message *m,
     const void * bytes_arr;
 
     /* Load all parameters */
-    r = sd_bus_message_read(m,
-                            "tuuyyuyy",
-                            &timestamp_ms,
-                            &src_addr,
-                            &dst_addr,
-                            &src_ep,
-                            &dst_ep,
-                            &travel_time,
-                            &qos,
-                            &hop_count);
+    r = sd_bus_message_read(
```

Comment:
In this case, could you remove clang formatting?
All those parameters are more readable with one on each line

Reply:
It actually makes no sense shy the format is being applied like this :(